xml/osc_building.xml

```diff
@@ -196,5 +195,192 @@ The buildroot was: /var/tmp/build-root/openSUSE_Tumbleweed-x86_64</screen>
   <remark>toms 2017-08-18: integrate content from  obs_build_preinstall.xml;
   using a _preinstallimage file</remark>
  </sect1>
- 
+
+ <sect1 xml:id="sec.obs.building.appimage">
+  <title>As &obs.ai;</title>
+  <para>
+   To create an &obs.ai;, you need:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     an <filename>appimage.yml</filename> file which acts as a build recipe.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     everything else you need from building a RPM package (see <xref
+      linkend="sec.obs.building.spec2rpm"/>).
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     a <filename>_service</filename> file
+     <remark>toms 2017-09-04: is this really needed? Or is it optional?</remark>
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     some configurations in your &obsa; project, see below.
+    </para>
+   </listitem>
+  </itemizedlist>
+  <para>
+   There are two ways to build an &obs.ai;:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>From RPM packages, or</para>
+   </listitem>
+   <listitem>
+    <para>directly from source code.</para>
+   </listitem>
+  </itemizedlist>
+  <para>To build a new &obs.ai; from RPM packages, proceed as follows:</para>
+  <procedure>
+   <title>Building &obs.ai; from RPM Packages</title>
+   <step>
+    <para>
+     Start with building RPM packages as described in <xref
+      linkend="sec.obs.building.spec2rpm"/>. Make sure the packages has
+     been built successfully.
+    </para>
+   </step>
+   <step>
+    <para>
+     Prepare your project to support &obs.ai;. In this example, we
+     use the home project <uri>&obshome1;</uri>:
+    </para>
+    <substeps>
+     <step>
+      <para>
+       To open the meta configuration, run:
+      </para>
+      <screen>&prompt.user;<command>osc</command> meta prj -e &obshome1;</screen>
+      <para>
+       <remark>toms 2017-09-05: Maybe this step should be explained in
+       a separate section/procedure?</remark>
```

Comment:
you're right, it is a bit distracting to have the same thing twice but using a different UI here.